tests/rule_based_profiler/data_assistant/test_data_assistant_runner.py

```diff
@@ -0,0 +1,102 @@
+import pandas as pd
+import pytest
+
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.data_context import DataContext
+from great_expectations.rule_based_profiler.data_assistant_result import (
+    DataAssistantResult,
+)
+from great_expectations.rule_based_profiler.domain import MetricDomainTypes
+
+
+@pytest.mark.integration
+def test_onboarding_data_assistant_runner_top_level_kwargs_allowed(
+    bobby_columnar_table_multi_batch_probabilistic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_probabilistic_data_context
+    batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+    try:
+        context.assistants.onboarding.run(
+            batch_request=batch_request,
+            include_column_names=["passenger_count"],
+            exclude_column_names=["tpep_pickup_datetime"],
+            include_column_name_suffixes=["amount"],
+            exclude_column_name_suffixes=["ID"],
+            cardinality_limit_mode="very_few",
+        )
+    except Exception as exc:
+        assert (False, f"context.assistants.onboarding.run raise an exception '{exc}'")
+
+    with pytest.raises(TypeError) as e:
+        context.assistants.onboarding.run(
+            batch_request=batch_request, non_existent_parameter="break_this"
+        )
+    assert (
+        e.value.args[0]
+        == "run() got an unexpected keyword argument 'non_existent_parameter'"
+    )
```

Comment:
You can do this in one go by making `pytest.raises()` matching.

Reply:
I am adding this to this PR: https://github.com/great-expectations/great_expectations/pull/6091